src/ConfigData/TableConfigurations/ColumnData.cs

```diff
@@ -215,14 +215,19 @@ public virtual void ReadBaseDataFromJSON(dynamic jsonConfig)
             {
                 LibraryID = libid;
             }
-
-            if ((jsonConfig?.qAttributeExpressions?.Count ?? 0) > 0)
-            {
-                BackgroundColorExpression = jsonConfig?.qAttributeExpressions[0]?.qExpression ?? "";
-            }
-            if ((jsonConfig?.qAttributeExpressions?.Count ?? 0) > 1)
+            if (jsonConfig?.qAttributeExpressions != null)
             {
-                TextColorExpression = jsonConfig?.qAttributeExpressions[1]?.qExpression ?? "";
+                foreach (dynamic attrExpression in jsonConfig?.qAttributeExpressions)
+                {
+                    if ((attrExpression?.id ?? "") == "cellForegroundColor")
+                    {
+                        TextColorExpression = attrExpression?.qExpression ?? "";
+                    }
+                    else
```

Comment:
this else path is not correct, Please use a switch for cellForeGround and CellBackground color and for default just do nothing.
Qlik kan introduce new id? and after that you can not just use this as new bakcground

Reply:
unfortunately there is no background - id

```
 "qAttributeExpressions":[
          {
            "id":"cellForegroundColor",
            "qExpression":"Yellow()"
          },
          {
            "qExpression":"Green()"
          }],
```

Reply:
If i look at the table properties with options/developer, both colors have an id asigned...
How can i get this (real) properties?

Reply:
Hm...  when i remove ``` DefaultValueHandling = Newtonsoft.Json.DefaultValueHandling.Ignore``` in the JsonSerializerSettings i get both ids...
```
"qAttributeExpressions": [
          {
            "id": "cellBackgroundColor",
            "qExpression": "Yellow()",
            "qAttribute": false
          },
          {
            "id": "cellForegroundColor",
            "qExpression": "Green()",
            "qAttribute": false
          }
```

Reply:
Ahhh, ok this field is an enum an background id its defaultproperty...
```
 public enum AttrId
    {
        CellBackgroundColor = 0,
        CellForegroundColor = 1,
        ColorByAlternative = 2,
        ColorByExpression = 3
    }
```

Reply:
@ChristofHoldschick yes, you are right with default handling ignore.
Can you fix it now, or do you still have an issue

Reply:
There's No issue left, but the time issue :-)
I will fix it tomorrow morning.